psamm/datasource/sbml.py

```diff
@@ -807,6 +826,12 @@ def write_model(self, file, model):
             reaction_tag.set(self._sbml_tag('reversible'), text_type(
                 equation.direction == Direction.Both).lower())
 
+            lower_str, upper_str = self._get_flux_bounds(
+                eq_id, model, flux_limits, equation)
+
+            reaction_tag.set(self._sbml_tag('lowerFluxBound'), lower_str)
+            reaction_tag.set(self._sbml_tag('upperFluxBound'), upper_str)
```

Comment:
Now that I look at this again I start to think that this is incorrect according to the FBCv2 spec. The `lowerFluxBound` and `upperFluxBound` should not contain the actual values but should instead contain references to the parameters defined below. Can you comment on that?